components/gallery/GalleryItem.vue

```diff
@@ -315,7 +321,15 @@ $break-point-width: 930px;
     }
   }
 }
-
+.back-button {
+  position: absolute;
+  left: 0.75rem;
+  top: 2rem;
+  z-index: 99;
```

Comment:
why this z-index?

Reply:
I need to set the z-index to be able to overlay on top of the global image, but it doesn't need to be as big as 99. Setting it to 1 works well enough